src/BlockReportSearch/Block/BlockUsageApi.php

```diff
@@ -51,6 +51,21 @@ public function get_count(): array
             $this->fetch_items();
         }
 
+        $posts_list_block = $this->usage->filter_query_blocks($this->items, $this->usage::POSTS_LIST_NAME);
+        $actions_list_block = $this->usage->filter_query_blocks($this->items, $this->usage::ACTIONS_LIST_NAME);
+
+        // Merge updated Posts List array with items array for API
+        if (!empty($posts_list_block)) {
```

Comment:
This code is the same as the one below for Actions List, maybe it could be shortened by using a `for` loop?